evalml/pipelines/component_graph.py

```diff
@@ -366,6 +366,14 @@ def predict(self, X):
         )
         return infer_feature_types(outputs.get(f"{final_component}.x"))
 
+    def _return_non_engineered_features(self, X):
+        base_features = [
+            c
+            for c in X.ww.columns
+            if X.ww[c].ww.origin == "base" or X.ww[c].ww.origin is None
```

Comment:
yup! Featuretools will tag engineered features as "engineered" and "base" otherwise. https://featuretools.alteryx.com/en/latest/resources/transition_to_ft_v1.0.html#Changes-to-Deep-Feature-Synthesis-and-Calculate-Feature-Matrix